js/components/ViewPagerIndicator.js

```diff
@@ -0,0 +1,84 @@
+import React, { PropTypes } from 'react';
+import {
+  View,
+  Animated,
+  Dimensions,
+  StyleSheet,
+} from 'react-native';
+
+const { width } = Dimensions.get('window');
+const DOT_SIZE = 12;
+const DOT_SPACING = DOT_SIZE / 2;
+const DOT_FULL_WIDTH = DOT_SIZE + DOT_SPACING * 2;
+
+/**
+ * This component is used to style the dots (or bullets) used by
+ * react-native-viewpager. It's a purely stateless functional component that is
+ * only concerned with rendering the dots and transitioning the active dot from
+ * one page of the view pager to another.
+ *
+ * @param pageCount {number} The total number of pages.
+ * @param activePage {number} The currently active page.
+ * @param scrollValue {Animated.Value} Used to transition the active dot.
+ * @param scrollOffset {number} Used to reset dot position after a transition.
+ * @return {Component}
+ */
+const ViewPagerIndicator = ({
```

Comment:
I understand this a shorthand declaration, but I'd rather we just be consistent with how we define components (ie. `class`).